examples/upload-media-from-local/README.md

```diff
@@ -0,0 +1,35 @@
+# Upload media from local
+
+This is a script to handle uploading media locally to Studio.
+
+## How to use
+
+Before starting using this script don't forget to authorize your request. this is explained here: [Authorization](../../README.md#authorization)
+
+For using this script we should run it through runner script in `bin` folder:
```

Comment:
I would remove this and also listing arguments. The former is explained in the root README, the latter is handled by argparse's help.

I would just have three examples:

To upload a single file:

```bash
bin/run examples/upload-media-from-local --subdomain <school_subdomain> /Users/some_user/Desktop/media/file.mp4
```

To upload multiple files:

```bash
bin/run examples/upload-media-from-local --subdomain <school_subdomain> /Users/some_user/Desktop/media/*
```

To upload on behalf of another user:
```bash
bin/run examples/upload-media-from-local --subdomain <school_subdomain> --user-id 3 /Users/some_user/Desktop/media/file.mp4
```

Reply:
done